Meta/gn/secondary/Userland/Libraries/LibWebView/BUILD.gn

```diff
@@ -44,6 +45,31 @@ compiled_action("WebDriverServerEndpoint") {
   ]
 }
 
+embed_as_string_view("generate_native_stylesheet_source") {
+  input = "Native.css"
+  output = "$target_gen_dir/NativeStyleSheetSource.cpp"
+  variable_name = "native_stylesheet_source"
+  namespace = "WebView"
+}
+
+source_set("all_generated") {
```

Comment:
I don't think we need this intermediate source set, we should be able to just add the generated sources to the LibWebView target -- which means we could just add the target gen dir include dir to that target as well.

Reply:
Alternatively, if you wanted to keep the source set, we should move all the Endpoint targets to it as well. since they are generated after all.

Reply:
Yep, true. I tried it first with just including the generate line as dep and it didn't work, because I forgot to add the source, and I misinterpreted the error.